### ISSUE TITLE
testDataSourceFromFiles 

### ISSUE THREAD

**Issue by sikandernoori** [SwiftyTesseract/SwiftyTesseract#96]
As per Documentation on how to use custom Location of traineddata files there should be a function named "testDataSourceFromFiles()" in  SwiftyTesseractTests.swift  explaining how to use custom location. But there isn't any.

Can you Please add that or provide details how to use this feature

```
let customDataSource = CustomDataSource()
let tesseract = SwiftyTesseract(
  language: .english, 
  dataSource: customDataSource, 
  engineMode: .lstmOnly
)
```
[See the testDataSourceFromFiles() test in SwiftyTesseractTests.swift (located near the end of the file) for an example on how this can be done.](https://cocoapods.org/pods/SwiftyTesseract#custom-location)

**Comment by sikandernoori** [comment]
[https://github.com/Minitour/SwiftyTesseract/blob/feec0bdac5a3dae3ce6db7d745d876121bd3693e/SwiftyTesseract/SwiftyTesseractTests/SwiftyTesseractTests.swift#L212](https://github.com/Minitour/SwiftyTesseract/blob/feec0bdac5a3dae3ce6db7d745d876121bd3693e/SwiftyTesseract/SwiftyTesseractTests/SwiftyTesseractTests.swift#L212)

This file needs merge in main

**Comment by Steven0351** [comment]
This file was merged, and was last available [in the 3.x.x version](https://github.com/SwiftyTesseract/SwiftyTesseract/blob/07a3be04dba2ee10975d2dcba689d3b56f9873ff/SwiftyTesseract/SwiftyTesseractTests/SwiftyTesseractTests.swift#L217). It looks like this was missed when I migrated the project to be a Swift package and added Linux support. If you want, you’re more than welcome to open a PR to fix the readme, I probably won’t be getting around to this before next week.
